socorro/cron/jobs/bugzilla.py

```diff
@@ -186,3 +179,52 @@ def _iterator(self, from_date):
                 int(report['id']),
                 find_signatures(report.get('cf_crash_signature', ''))
             )
+
+
+class BugzillaCronAppDryRunner(App):  # pragma: no cover
```

Comment:
Right! Good point. The reason I started writing this class was to be able to do a bit more detailed "manual" logging of exactly what SQL was being executed. Then I fell into a deep rabbit hole of trying to get mock to work. I eventually failed, took a step back to using a real postgres table. What I didn't realize was this very point you're making. That it's easier to just run it with crontabber_app.py

Will update.